server/routes/api/ads.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../../db');
+
+router.get('/', async (req, res) => {
```

Comment:
req: Request, res: Response

For future typescript fields access

Reply:
the server side is in javascript, We need to have a task to change it

Reply:
I accept 😃 , this is `MVP`.

I will add this motion in `trello`